pkg/common/tempfile.go

```diff
@@ -37,12 +37,12 @@ func reseed() uint32 {
 
 func nextPrefix() string {
 	randmu.Lock()
-	r := rand
+	r := randomNumber
 	if r == 0 {
 		r = reseed()
 	}
 	r = r*1664525 + 1013904223 // constants from Numerical Recipes
-	rand = r
+	randomNumber = r
```

Comment:
imo golang 1.14 tempfile generation is sufficient for this usecase. we can wrap it in retryUntilSuccessfull of course